### PR TITLE
create week 7 schedule

### PR DESCRIPTION
**Pull request by bo-bok**
**admin**
* adds week 7 folder to coursebook, containing week schedule 
* link to week 7 readme from main coursebook readme

**specific information added to week's readme**
* link to workshop 1 
* link to workshop 3 [WIP]
* Wednesday speaker info

relates #356

Note: this is all the current information, will add links to workshops/morning challenges when they are ready

### CODE REVIEW

File: coursebook/week-7/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+## Week Schedule and Responsibilities
+
+See [general week structure](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/general/week-schedule.md#general-week-schedule) for us to follow.
```
Comment (jsms90) [inline coursebook/week-7/README.md]: Careful. This was only a note within the issue, to help you guys. We don't need this in the actual file :smile:

File: coursebook/week-7/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Create this file and then update the wording and the links later, via the separate issues on each "part"
```
Comment (jsms90) [inline coursebook/week-7/README.md]: Same here

File: coursebook/week-7/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+#### Day 1
+
+- 10:00 - 10:15 - Introduction to learning outcomes
+
+- 10:15 - 13:00 - Workshop on [Password Management](https://github.com/foundersandcoders/ws-password-management)
+
+— LUNCH —
+
+- 14:00 - 16:00 - Workshop 2
```
Comment (jsms90) [inline coursebook/week-7/README.md]: @bo-bok @alexis-l8 @ronanyeah @des-des Do we still not know what this workshop is going to be on?

File: coursebook/week-7/README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+- 16:00 - 18:00 - Business development
+
+#### Day 2
+
+- 10:00-11:00 - Morning Challenge 1
+
+- 11:00-12:45 - Workshop on [Session Management](https://github.com/foundersandcoders/ws-session-management)[WIP]
```
Comment (jsms90) [inline coursebook/week-7/README.md]: Is this definitely workshop 3, not workshop 2 then?

File: coursebook/week-7/README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+- 10:00 - 11:00 - Morning Challenge 2
+
+- 11:00 - 13:00 - Projects (planning)
+
+— LUNCH —
+
+- 14:00 - 17:00 - Projects
+
+- 17:00 - 18:00 - External Speaker: Enrique Comba [8th Light](https://8thlight.com/)
```
Comment (jsms90) [inline coursebook/week-7/README.md]: Please don't hard code in specific speakers

**Comment by jsms90** [comment]
@ronanyeah @des-des I'm really concerned if we still don't even know what the content of the morning challenges and workshops are going to be. This is going to be delivered in one week. Perhaps you guys need to pull in other alumni to help you?

**Comment by ronanyeah** [comment]
@jsms90 It'll get done, no worries.

**Comment by jsms90** [comment]
@bo-bok Could you update this with the repo links, now that we have all the pieces? I'm eager to get this merged asap

**Pull request merged**